src/administration/install_docker.md

```diff
@@ -3,116 +3,45 @@
 Make sure you have both docker and docker-compose(>=`1.24.0`) installed. On Ubuntu, just run `apt install docker-compose docker.io`. Next,
 
 ```bash
-# create a folder for the lemmy files. the location doesnt matter, you can put this anywhere you want
+# Create a folder for the lemmy files. the location doesnt matter, you can put this anywhere you want
 mkdir /lemmy
 cd /lemmy
 
-# download default config files
-wget https://raw.githubusercontent.com/LemmyNet/lemmy/main/docker/docker-compose.yml
-wget https://raw.githubusercontent.com/LemmyNet/lemmy/main/docker/lemmy.hjson
+# Download default config files
+
+# These two files contain {{ }} braces for variables, such as passwords, and your domain.
+# Edit them before starting up lemmy for the first time.
+# The images will likely be: dessalines/lemmy:VERSION and dessalines/lemmy-ui:VERSION
```

Comment:
Include clickable links to docker hub instead.

Reply:
K will do.